full/Dockerfile

```diff
@@ -133,8 +133,9 @@ RUN curl -fsSL https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-ins
         echo 'eval "$(pyenv virtualenv-init -)"'; } >> .bashrc \
     && pyenv install 3.6.6 \
     && pyenv global 3.6.6 \
-    && pip install virtualenv python-language-server[all]==0.19.0 \
-    && rm -rf /tmp/*
+    && pip install virtualenv pipenv python-language-server[all]==0.19.0 \
+    && sudo apt-get update && sudo apt-get install -yq python-dev python3-dev \
```

Comment:
can you explain, why `python-dev` and `python3-dev` is necessary to be installed? 

it brings `header files and a static library for Python`, right? but if we already install python via `pyenv` (which is built from source,) is available as well. 

AFAIK, the very installation of the `python3` from ubtuntu is there just because it's required by something else.

I'm not sure, how this can be useful if `python3` points to the `pyenv`'s version.

Reply:
Well, `python-dev` does seem to be the solution to fix `fatal error: Python.h: No such file or directory`.

I speculatively also installed `python3-dev`, as [suggested on StackOverflow](https://stackoverflow.com/questions/21530577/fatal-error-python-h-no-such-file-or-directory), but you're probably right that it's not helpful if `python3` is only in `pyenv` (I don't know much about Python development).

I'll remove `python3-dev` as it doesn't fix an actual error I had.